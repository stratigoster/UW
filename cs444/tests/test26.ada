PACKAGE foo IS
  TYPE data;
  TYPE pdata IS access data; -- forward references work!
  TYPE data IS RECORD
    a : integer;
    b : boolean;
    p : pdata;
  END RECORD;
  type blah is array (integer) of integer;
  TYPE ar_data IS array (1) of pdata;
--                      ^^^
-- not a range; should be of the form 1..x where x is an integer
  some_data : ar_data;
END foo;
