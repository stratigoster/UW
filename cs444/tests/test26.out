***ERROR***:10:28:	Expression must be followed by .. and expression

------
Compilation error
------
